examples/exitwhat.fio

```diff
@@ -32,7 +32,7 @@ iodepth=32
 rate=300,300,300
 
 [slow2]
-stonewall=1
```

Comment:
If this change in job files is required, then we'll have to fix it so that's not the case. If it's not required, don't make the change in the job file.

Reply:
Stonewalls usage is not documented like this. I've changed the man page and also fixed some other stuff in the example. Now it's up to you if we should use `stonewall=1` in the example or not. Both will work and it's not required to set a value. But all other examples just use stonewall without any value assigned.